Test branch

Mikhail-Vokhmianin opened the pull request.

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Update example_script.py

Mikhail-Vokhmianin committed:
Update example_script.py

Mikhail-Vokhmianin committed:
Update setup.py

Mikhail-Vokhmianin committed:
Create

Mikhail-Vokhmianin committed:
Update setup.py

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Update example_script.py

bellatrics commented on suomi/setup.py:
```diff
@@ -4,9 +4,9 @@
       version = "0.1.0",
       packages = find_packages(include=["suomi","suomi.*"]),
-      install_requires = ['datetime', 'random'],
+      install_requires = ['datetime', 'random','requests'],
```
many places where formatting needs some standardization. we'll discuss this later

bellatrics commented on README.md:
```diff
@@ -6,4 +6,10 @@ A repository made for spending time coding and commiting! **Yay**!
 ## Setup
 pip install -e .
```
put this into "code-style" using markup language

bellatrics commented on README.md:
```diff
@@ -6,4 +6,10 @@ A repository made for spending time coding and commiting! **Yay**!
 
+## Usage
 
+from suomi import greets
```
in the package there should be relative imports (we'll discuss this later)

bellatrics commented on example_script.py:
```diff
@@ -1,3 +1,11 @@
 
-greets()
\ No newline at end of file
+greets('World')
```
don't forget to add a test

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Update setup.py

Clear requirements for the `datetime` and `random` packages as they are built-in python by default

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Update example_script.py

Mikhail-Vokhmianin committed:
Add files via upload

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Update README.md

Mikhail-Vokhmianin committed:
Delete kotkin_comments.csv

Mikhail-Vokhmianin committed:
Delete stone_comments.csv

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

# Conflicts:
#	examples/example_script.py

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

# Conflicts:
#	examples/example_script.py

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch

Mikhail-Vokhmianin committed:
Merge remote-tracking branch 'origin/test_branch' into test_branch
